app/scripts/TiledPlot.js

```diff
@@ -484,14 +486,14 @@ class TiledPlot extends React.Component {
 
     // Reduce the list of tracks to a dictionary of track ids. This is useful
     // to speedup the subsequent filtering
-    const trackUids = allTracksWithTilesetInfos.reduce((a, b) => {
-      a[b.id] = true;
-      return a;
-    }, {});
-
-    // Only could tracks that are suppose to get a tileset
-    const loadedTilesetInfos = Object.keys(this.tracksByUidInit).filter(
-      trackUid => trackUids[trackUid]
+    // const trackUids = allTracksWithTilesetInfos.reduce((a, b) => {
```

Comment:
Is there a reason to keep this commented rather than remove?

Reply:
Nope.